packages/react/src/lib/handleRef.ts

```diff
@@ -21,9 +21,8 @@ const handleRef = <N>(ref: React.Ref<N>, node: N) => {
   }
 
   if (ref !== null && typeof ref === 'object') {
-    // @ts-ignore The `current` property is defined as readonly, however it's a valid way because
-    // `ref` is a mutable object
-    ref.current = node
+    // The `current` property is defined as readonly, however it's a valid way because `ref` is a mutable object
+    ;(ref as React.MutableRefObject<N>).current = node
```

Comment:
Is the `;` cruft?

Reply:
Nope and this is sad: https://github.com/prettier/prettier/issues/4630 😭

Reply:
![image](https://user-images.githubusercontent.com/14183168/56139092-d4234c80-5fa0-11e9-8e9e-165e9a4b5d68.png)
https://stardust-ui.github.io/react/shorthand-props
The same case, actually 😨

Reply:
Ok, just to mention I hate this, especially that we are not using `;` at all... Sad day for me :D

Reply:
Even worse with the provided example, it is used in our guides... :\